res/qml/main.qml

```diff
@@ -90,132 +88,152 @@ ApplicationWindow {
 
                     DeveloperToolsWindow {
                         id: devToolsWindow
-
-                        width: 640
                         height: 480
+                        width: 640
+                    }
+                }
+                Skin.Button {
+                    id: showPreferencesButton
+                    activeColor: Theme.white
+                    checked: settingsPopup.opened
+                    icon.height: 16
+                    icon.source: "images/gear.svg"
+                    icon.width: 16
+                    implicitWidth: implicitHeight
+
+                    onClicked: {
+                        if (!settingsPopup.opened) {
+                            settingsPopup.open();
+                        }
+                    }
+                    onPressAndHold: {
+                        Mixxx.PreferencesDialog.show();
                     }
```

Comment:
just to clarify, this shows the placeholder QML dialog on click and the fully featured widget-based one on press and hold as a fallback?